### PR TITLE
🌐 [i18n-KO] docs: ko: Chap3 Transformer architectures for audio

### PR DESCRIPTION
**Pull request by gabrielwithappy**
# What does this PR do?
Translated the multiple_choice.mdx file of the documentation to Korean.
Feel free to discuss about this PR on [discord channel](https://discord.com/channels/879548962464493619/1105153909552586812)
Thank you in advance for your review.

Part of https://github.com/huggingface/audio-transformers-course/issues/67

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. [[lowercased-header]])
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)
@jungnerd @jinnsp @wonhyeongseo

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/audio-transformers-course#%EF%B8%8F-contributing-a-new-chapter), Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? or [discord](https://discord.com/channels/879548962464493619/1105153909552586812) Please add a link
to it if that's the case.
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

May you please review this PR? 
@MKhalusova

### CODE REVIEW

**Comment by gabrielwithappy** [comment]
@jungnerd @jinnsp @wonhyeongseo
Please review this PR :-)

**Comment by MKhalusova** [comment]
Is the PR ready for review?

**Comment by gabrielwithappy** [comment]
> Is the PR ready for review?

The job is done. I am waiting for reviews from Korean reviewers. Though I am also asking to huggingface discord channel, no reviews so far. 😭
If there is no problem, I would appreciate it if you could review or run a live preview 🙂

**Comment by gabrielwithappy** [comment]
fix some toc title and ambiguous sentences

**Comment by gabrielwithappy** [comment]
Dear @MKhalusova,
Thank you for a live preview
I found my mistakes and fix some toc title and ambiguous sentences.
Please review this PR :-)
BRs

File: chapters/ko/chapter3/quiz.mdx [hunk 1]
```diff
@@ -0,0 +1,139 @@
+<!-- DISABLE-FRONTMATTER-SECTIONS -->
+
+# 코스 자료에 대한 이해도를 확인해보세요[[check-your-understanding-of-the-course-material]]
```
Comment (gabrielwithappy) [inline chapters/ko/chapter3/quiz.mdx:3]:
```suggestion
# 이번 코스에 대한 이해도를 확인해보세요[[check-your-understanding-of-the-course-material]]
```

**Review by gabrielwithappy — COMMENTED** [review]
refine korean sentenses

File: chapters/ko/chapter3/introduction.mdx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+이 강좌에서 다룰 오디오 모델은 일반적으로 위와 같은 표준 트랜스포머 아키텍처를 사용하지만, 텍스트 대신 오디오 데이터를 사용할 수 있도록 입력 또는 출력 측에서 약간의 수정이 이루어집니다. 이러한 모든 모델은 기본적으로 트랜스포머이므로 대부분의 아키텍처가 공통적이며 주요 차이점은 학습 및 사용 방식에 있습니다.
+
+<div class="flex justify-center">
+    <img src="https://huggingface.co/datasets/huggingface-course/audio-course-images/resolve/main/transformers_blocks.png" alt="The transformer with audio input and output">
+</div>
+
+오디오 작업의 경우 입력과/또는 출력 시퀀스가 텍스트가 아닌 오디오일 수 있습니다:
+
+- 자동 음성 인식(ASR): 입력은 음성, 출력은 텍스트입니다.
```
[the remark this answers is not in the document]
Comment (gabrielwithappy) [reply chapters/ko/chapter3/introduction.mdx:33]: done

**Comment by gabrielwithappy** [comment]
- refine korean sentenses
- add abbreviation description

**Comment by gabrielwithappy** [comment]
Dear @MKhalusova,
Please reivew this PR and run a live preview :-)
thank you in advance.
BRs.

**Comment by wonhyeongseo** [comment]
Hello @gabrielwithappy , I believe this PR's live-preview from the [2nd comment](https://moon-ci-docs.huggingface.co/learn/audio-course/pr_94) is still alive! 😺

**Comment by gabrielwithappy** [comment]
> Hello @gabrielwithappy , I believe this PR's live-preview from the [2nd comment](https://moon-ci-docs.huggingface.co/learn/audio-course/pr_94) is still alive! 😺 

Thx a million 🤩

**Comment by gabrielwithappy** [comment]
Dear @MKhalusova,
Please reivew this PR
thank you in advance.
BRs.

File: chapters/ko/_toctree.yml [hunk 3]
```diff
@@ -40,3 +40,19 @@
   sections:
   - local: events/introduction
     title: 라이브 세션과 워크샵
+
```
Comment (MKhalusova) [inline chapters/ko/_toctree.yml:43]: Please move the unit 3 before the events section. The events section is supposed to be at the end of the TOC.

**Review by MKhalusova — COMMENTED** [review]
Thank you for your work! It looks great to me, only one small improvement is required in order to merge. The order in the TOC is slightly off, please see my comment

File: chapters/ko/_toctree.yml (hunk 3 above)
Comment (gabrielwithappy) [reply chapters/ko/_toctree.yml:43]: I got it.

**Pull request merged**